bedrock/settings/__init__.py

```diff
@@ -258,15 +259,21 @@
         "connect-src": list(set(_csp_default_src + _csp_connect_src)),
         # support older browsers (mainly Safari)
         "frame-src": _csp_child_src,
-        "report-uri": config("CSP_REPORT_URI", default="") or None,
     },
 }
 
+# Start report-only CSP as a copy. We'll modify it later if needed.
+# Only set up report-only CSP if we have a report-uri set.
+if csp_report_uri := config("CSP_REPORT_URI", default="") or None:
```

Comment:
Is the `or None` needed here? The default of `""` should be falsey, no?

Reply:
django-csp treats all directives that are explicitly `None` as unset. So yeah an empty string will make it not end up in the header, but I'm wanting it more explicit that it's not meant to be set.